index.html

```diff
@@ -205,6 +250,9 @@ <h3>
                 </div>
             </form>
 
+        </section>
+
+        <footer>
             <ul class="socialLinks">
```

Comment:
- [ ] OPTIONAL: The social links element has a gray line on top of it. Kindly implement it in your spare time. 
![image](https://user-images.githubusercontent.com/64894927/121436085-98218180-c988-11eb-9773-d0e51c27e467.png)